assets/js/admin/form-builder.js

```diff
@@ -222,6 +223,290 @@
+			var $field_options_container = $( '#everest-forms-field-option-' + field_id );
+
+			// Initialize color picker for Handle Color option.
+			$field_options_container.find( '.evf-range-slider-handle-color' )
+			.wpColorPicker({
+				change: function( event, ui ) {
+					var new_color = $( event.target ).val();
+					var field_id = $( this ).closest( '.everest-forms-field-option-row' ).data( 'field-id' );
+					var current_skin = $field_options_container.find( '.evf-range-slider-skin' ).val();
+
+					EVFPanelBuilder.setSliderHandleColor( field_id, new_color, current_skin );
+				}
+			});
+		},
+
+		/**
+		 * Initialize Range Slider Highlight Color Picker.
+		 *
+		 * @since 1.7.0
+		 */
+		initializeSliderHighlightColorOption: function( field_id ) {
+			var $field = $( '#everest-forms-field-' + field_id );
+			var $field_options_container = $( '#everest-forms-field-option-' + field_id );
+
+			$field_options_container.find( '.evf-range-slider-highlight-color' )
+			.wpColorPicker({
+				change: function( event, ui ) {
+					var new_color = $( event.target ).val();
+
+					if ( new_color ) {
+						$field.find( '.irs-bar' ).css( 'background', new_color );
+					}
+				}
+			});
+		},
+
+		/**
+		 * Initialize Range Slider Track Color Picker.
+		 *
+		 * @since 1.7.0
+		 */
+		initializeSliderTrackColorOption: function( field_id ) {
+			var $field = $( '#everest-forms-field-' + field_id );
+			var $field_options_container = $( '#everest-forms-field-option-' + field_id );
+
+			$field_options_container.find( '.evf-range-slider-track-color' )
+			.wpColorPicker({
+				change: function( event, ui ) {
+					var new_color = $( event.target ).val();
+
+					if ( new_color ) {
+						$field.find( '.irs-line' ).css( 'background', new_color );
+					}
+				}
+			});
+		},
+
+		/**
+		 * Update a Range Slider field's Handle/Highlight/Track colors.
+		 *
+		 * @since 1.7.0
+		 */
+		updateRangeSliderColors: function ( field_id ) {
+			var $field = $( '#everest-forms-field-' + field_id );
+			var $field_options_container = $( '#everest-forms-field-option-' + field_id );
+
+			// Set handle color for the Slider field.
+			var handle_color = $field_options_container.find( '.evf-range-slider-handle-color' ).val();
+			var skin = $field_options_container.find( '.evf-range-slider-skin' ).val();
+			EVFPanelBuilder.setSliderHandleColor( field_id, handle_color, skin );
+
+			// Set Current Highlight Color.
+			var highlight_color = $field_options_container.find( '.evf-range-slider-highlight-color' ).val();
+			$field.find( '.irs-bar' ).css( 'background', highlight_color );
+
+			// Set Current Track Color.
+			var track_color = $field_options_container.find( '.evf-range-slider-track-color' ).val();
+			$field.find( '.irs-line' ).css( 'background', track_color );
+		},
+
+		/**
+		 * Set a Range Slider's handle color.
+		 *
+		 * @since 1.7.0
+		 */
+		setSliderHandleColor: function ( field_id, color, skin ) {
+			if ( '' !== field_id && color && skin ) {
```

Comment:
Comparision isn't strict with color and skin. Please explicit the comparison, as comparisons are malleable in js.

Reply:
Resolved with 38e1bd4fdd984eab4133668204c60ca1ca725b73